patches/strict-popup-blocking.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/components/blocked_content/popup_blocker.cc b/components/blocked_content/popup_blocker.cc
+index 8b12c7dd441f3..9c13ac10a6f1a 100644
+--- a/components/blocked_content/popup_blocker.cc
++++ b/components/blocked_content/popup_blocker.cc
+@@ -81,11 +81,8 @@ PopupBlockType ShouldBlockPopup(content::WebContents* web_contents,
+     return PopupBlockType::kNotBlocked;
+   }
+ 
+-  auto* safe_browsing_blocker =
```

Comment:
this won't work I don't think

```

  if (!user_gesture)
    return PopupBlockType::kNoGesture;
```

this code is above it, and this is the core issue